src/components/twilio-voice-basic-call-control/public/twilio-voice-basic-call-control.js

```diff
@@ -22,75 +19,165 @@ class TwilioVoiceBasicCallControl extends HTMLElement {
     });
 
     this.shadowRoot
-      .querySelector('#hold')
-      .addEventListener('click', () => this.#handleHold(true));
-    this.shadowRoot
-      .querySelector('#resume')
-      .addEventListener('click', () => this.#handleHold(false));
-    this.shadowRoot
-      .querySelector('#mute')
-      .addEventListener('click', () => this.#handleMute(true));
-    this.shadowRoot
-      .querySelector('#unmute')
-      .addEventListener('click', () => this.#handleMute(false));
+      .querySelector('#add-conference')
+      .addEventListener('click', () => this.#handleAddConference());
+  }
+
+  async #handleAddConference() {
+    const participant = this.shadowRoot.querySelector('#add-participant').value;
+
+    if (participant === '') {
+      console.log('Please enter a validparticipant');
```

Comment:
```suggestion
      console.log('Please enter a valid participant');
```

Reply:
I also prefer to just `return` on failure here, that way we don't need an `else` block and the code is less nested.

```
if (participant === '') {
  console.error('Please enter a valid participant'); // consider throwing instead? what's the UX if an error is thrown here? does the app crash, or just log an error?
  return;
}
```

Reply:
The backend does crash here, so will throw an error